product_customer_code/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+    
+======================
+Products Customer Code
+======================
+
+This module does just like the product.supplierinfo but for customers instead. For instance it
+allows to have different references for the same product according to the customer.
+
+.. image:: product_customer_code/static/src/img/screenshot1.png
+
+.. tip::
+
+    You will need install some of the Apps which enable the product menu to see this module in
+    action, like Sales, Purchase or Warehouse Management
+
+Installation
+============
+
+To install this module, you need to:
+
+1.  Clone the branch 8.0 of the repository https://github.com/OCA/product-attribute
+2.  Add the path to this repository in your configuration (addons-path)
+3.  Update the module list
+4.  Go to menu *Setting -> Modules -> Local Modules*
+5.  Search For *Products Customer Code*
+6.  Install the module
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/8.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/product-attribute/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+product-attribute/issues/new?body=module:%20
+product_customer_code%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
Please use the new bug tracker section like at https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst